src/main/java/core/basesyntax/model/FruitsForSale.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.model;
+
+import java.util.List;
+
+public class FruitsForSale {
```

Comment:
We can work with any fruits, banana and apple just an example, so we don't need this class